rmw_fastrtps_cpp/CMakeLists.txt

```diff
@@ -53,7 +54,7 @@ ament_target_dependencies(rmw_fastrtps_cpp
 
 configure_rmw_library(rmw_fastrtps_cpp)
 
-ament_export_libraries(rmw_fastrtps_cpp fastcdr fastrtps)
+ament_export_libraries(rmw_fastrtps_cpp fastcdr fastrtps ${FastRTPS_LIBRARIES})
```

Comment:
The changes in this file aren't needed anymore when ament/ament_cmake#81 is merged.